modules/core/client/config/core.client.menus.js

```diff
@@ -0,0 +1,48 @@
+(function () {
+  'use strict';
+
+  // Configuring the Fishing module
+  angular
+  .module('core')
+  .run(MenuConfig);
+
+  MenuConfig.$inject = ['Menus'];
+
+  function MenuConfig(Menus) {
+
+    Menus.addMenu('account', {
+      roles: ['user']
+    });
+
+    Menus.addMenuItem('account', {
+      title: '',
+      state: 'settings',
+      type: 'dropdown',
+      roles: ['user']
+    });
+
```

Comment:
I've always been confused by the implementation of the Menus service. It never seems intuitive to me.

When would we have multiple *Menu Items* defined for a Menu?
One menu defined with?
&nbsp; Item1
&nbsp; &nbsp; SubItem1
&nbsp; &nbsp; SubItem2
&nbsp; Item2
&nbsp; &nbsp; SubItemA
&nbsp; &nbsp; SubItemB

Makes sense to me for the Account menu:
Edit Profile
Edit Profile Pic
Manage Social Account

It seems that the options defined in the MenuItem should be defined at the Menu level. Maybe I'm looking at all this wrong? Like I said, every time I look at this service it confuses me :)